### PR TITLE
Refactors feedback form.

### LINKED ISSUE #766: Refactor: Rewrite feedback form JS as a stimulus controller (without jQuery)
Once this is rewritten without jQuery, jQuery can be removed.

### PR DESCRIPTION
**Pull request by justinlittman**
* Uses client-side validation
* Gets rid of jquery
* Cleans up views on the assumption that only single use of feedback form

closes #766

### CODE REVIEW

File: app/controllers/feedback_forms_controller.rb [hunk 1]
```diff
@@ -1,49 +1,22 @@
...
-      format.html do
-        redirect_to params[:url]
-      end
-    end
+    redirect_to params[:url]
   end
 
   protected
 
-  def validate
```
Comment (justinlittman) [inline app/controllers/feedback_forms_controller.rb:36]: This validation is now client-side.

File: app/javascript/application.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "controllers"
 import 'bootstrap';
-import 'feedback_form';
```
Comment (justinlittman) [inline app/javascript/application.js:4]: Goodbye.

File: app/javascript/controllers/feedback_controller.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static targets = ['form']
+
+  connect() {
```
Comment (justinlittman) [inline app/javascript/controllers/feedback_controller.js:6]: This is the only remaining JS that is necessary.

File: app/views/feedback_forms/new.html.erb [hunk 4]
```diff
@@ -1 +1,53 @@
-<%= render 'shared/feedback_forms/form' %>
```
Comment (justinlittman) [inline app/views/feedback_forms/new.html.erb:1]: Merged the many feedback form partials.

File: config/importmap.rb [hunk 5]
```diff
@@ -1,7 +1,6 @@
 # Pin npm packages by running ./bin/importmap
 
 pin "application", preload: true
-pin "jquery", to: "https://ga.jspm.io/npm:jquery@3.6.1/dist/jquery.js"
```
Comment (justinlittman) [inline config/importmap.rb:4]: Goodbye.

**Comment by justinlittman** [comment]
@amyehodge approved.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**